scalyr_agent/copying_manager.py

```diff
@@ -1031,7 +1033,7 @@ def __read_checkpoint_state(self):
             # if the active checkpoint file is newer, overwrite any checkpoint values with the
             # updated full checkpoint
             active_checkpoints = scalyr_util.read_file_as_json(
-                active_checkpoint_file_path
+                active_checkpoint_file_path, strict_utf8=True
```

Comment:
Can you add a little more to the error logged at line 1048.  Have it say: `Could not read checkpoint file due to error.  Ignoring checkpoint file.`